client-projects/the-lumery/scss/nav.scss

```diff
@@ -55,6 +55,7 @@
 
 .nav__dropdown-list.w--open {
   border-top-color: var(--color-black);
+  margin-top: 0;
```

Comment:
bizarrely there's a `margin-top: .5rem` setting on this class combo (you don't have access to `.w--open` from within webflow, you can only change it with custom code). 
If you make _some particular combination_ of layout changes, this will open up a gap between your toggle and your dropdown area, so when your mouse crosses that gap, it has left the dropdown, and the dropdown content will disappear. Magic! 🪄🎩🐇